smtp/smtpc.c

```diff
@@ -158,9 +158,9 @@ smtpc_init(void)
 
 	const char *libname;
 #ifdef __APPLE__
-	libname = "libcurl.dylib";
+	libname = "libcurl.4.dylib";
```

Comment:
Regarding Mac OS: the system library is versioned (has `.4`) and the library from homebrew as well, correct?

Reply:
Yes, that's right.

Reply:
Okay so.